rfcs/0074-community-coordination-hub.md

```diff
@@ -0,0 +1,226 @@
+media, capital, labor and other ways. We need to better leverage our community
+resources (tools, knowledge, people).</sub>
+
+# Detailed design
+[design]: #detailed-design
+
+A repository shall exist, named `NixOS/community`, which will act as central hub
+for coordinating the community's efforts.
+
+It shall make it easy to mentally grasp how the community coordinates as a whole.
+Getting involved with the community should feel as natural as skimming through
+code on github (or on a local clone).
+
+It shall be a place where established truths or conclusions that (have) emerge
+from the communities conversations and decision making processes are recorded.
+
+Therefore it shall act as single source of truth for aspects of:
+
+- Communication
+  - Channels
+  - Modes & Policies
+- Governance
+  - Processes
+  - Structure
+- Onboarding
+  - Code Contributions
+  - Community Engagement
+
+It shall _not_:
+
+- document the standard library (or any other highly specific and established technical subject).
+- act as discussion forum (but document outcomes).
+- implement code (but coordinate it's implementation).
+- exhibit established content that best fits into the manual (but call out to it and coordinate working drafts or ideas about it).
+
+To implement, myself, and anyone interested, will review https://github.com/kubernetes/community
+to the smallest details.
+
+Based on this review, we'll create a clean new repository at
+`NixOS/community`, with what we considered was worth porting from
+[Kubernetes' community portal](https://github.com/kubernetes/community).
+
+As a general guideline for this review, we'd acknowledge the success
+and impressive dynamics of the Kubernetes ecosystem.
+
+## Brainstorm
+[brainstorm]: #brainstorm
+
+This section exposes loose ideas we might put into `NixOS/community` collected during
+this RFC's review process.
+
+- Commnication/Channels: [Community Event Calender](https://github.com/kubernetes/community/tree/master/events)
+- Governance/Processes: Community Event Planning
+- Onboarding/Code Contributions: Explain and Support an Efficient Code Review Process.
+- Onboarding/Community Engagement: Move non-technical aspects of nixpkgs' manual here
+- Governance/Processes/Transparency: Record Meeting Minutes of working groups
+- Governance/Structure: What are the structures about which the community behaves and coordinates?
+- Governance/Structure/Acting Bodies: Evolve Design Prototypes before moving to RFC with in special interest groups
+- Governance/Processes: How does the Community make decisions (formal and informal)?
+- Governance/Processes: How disagreement are resolved (given consensus about a need is established)?
+- Onboarding/Code Contributions: How to quickstart and where to find detailed (authoritative resoures)?
+- Onboarding/Community Engagment: Don't be scared about asking stupid questions, unless you are aware they are stupid!
```

Comment:
I tend to think all of this information can belong elsewhere:

- Announcements for events show up nice at discourse.
- Onboarding related issues should be in the WiKi.

I don't know if Governance processes and structures should be so strict and binding, but I'm not strongly opinionated. @blaggacao you wrote:

> To implement, myself, and anyone interested, will review https://github.com/kubernetes/community
to the smallest details.

Maybe you and anyone interested should review this portal now, and thicken this RFC with even more details and ideas? I'm still not convinced this community portal will fit our community - it's still a different thing, Kubernetes vs an OS.

Reply:
Now it's the other way round :wink: : I don't see before my inner eye what you are seeing.

I'm reasoning pretty liniear and I'm not even reaching too far:
- Motivation: make better use of resources
- Solution: a _centralized_ "Coordination Hub" (can it be more prominent rhan in the title?)

I don't we can achieve this see when there is not even central entrypoint. I think I'll see in a clarifying commit how to fence against an alternative interpretion that we can just distribute things (and actually stow them away) into multiple artefacts.

I thibk before moving forward foo quickly, we need to sort out two things:
- Potential Legitimacy Chain (esp. for tings like working groups, that may have limited authonomy)
- Agree about the motivation and the General Design

As I said before, the second point is very important to me so that I can find the confidence to spend more time in working out a really good and detailed solution to the problem.

Reply:
We could organize a public poll some time from now and record the results back into the RFC... :smile: I would also recognize it as a strong signal. :party:

Reply:
Are you ok with postponing a more detailed specification upon obtaining clear signals / a mandate, as proposed in https://github.com/NixOS/rfcs/pull/74/commits/f55e7ff2f8b95f35132f1ff300a2829939d10921 ?

Reply:
Addressed in https://github.com/NixOS/rfcs/pull/74/commits/571e8ec8acdd6676bd61c84947f96835dca3284c - closing ok?